src/services/feed_manager.py

```
@@ -20,34 +20,36 @@
         self.feed_validator = FeedValidator(timeout=10)
-        
+
     async def load_opml(self) -> Tuple[int, Dict[str, str]]:
         """Load and parse the OPML file using listparser.
-        
+
         Returns:
             Tuple[int, Dict[str, str]]: Number of feeds loaded and dict of invalid feeds with errors
         """
         try:
             with open(self.opml_file) as f:
```

Ruff (0.8.0)

Async functions should not open files with blocking methods like `open`

(ASYNC230)